packages/web/src/pages/upload-page/fields/stream-availability/SpecialAccessFields.tsx

```diff
@@ -78,22 +77,22 @@ export const SpecialAccessFields = (props: TrackAvailabilityFieldsProps) => {
 
   return (
     <>
-      <RadioButtonGroup
+      <RadioGroup
```

Comment:
note, removing the "button" nomenclature for these elements to align with harmony spec